src/assets/styles/_forms.scss

```diff
@@ -58,10 +63,47 @@
   font-size: $kui-font-size-50;
 }
 
+.field-group-list > *+* {
+  margin-top: $kui-space-80;
+}
+
+.field-group-label {
+  display: inline-flex;
+  margin-bottom: $kui-space-40;
+  font-size: $kui-font-size-30;
+  line-height: $kui-line-height-30;
+  font-weight: $kui-font-weight-semibold;
+}
+
 .radio-button-group>* {
   display: flex;
 }
 
 .radio-button-group>*+* {
   margin-block-start: $kui-space-40;
 }
+
+.instruction-list >*+* {
+  margin-top: $kui-space-50;
+}
+
+.form-step-title {
+  font-size: $kui-font-size-60;
+}
+
+.form-step-title:not(:first-child) {
+  margin-top: $kui-space-90;
+}
+
+.form-step-number {
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  width: 30px;
+  height: 30px;
+  margin-right: $kui-space-20;
+  color: $kui-color-text-inverse;
+  background-color: #169fcc;
```

Comment:
I know we kinda spoke about this before, but what would you think of having `--kuma-blue-???` etc etc and using those for any random hex codes we have. Even if they are just one off instances to my eyes it makes things easier, plus it means we would have a central list of these random colors in a single place.

Reply:
I’ve moved this into a variable.